pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -53,12 +53,12 @@ final: prev:
 
   Coqtail = buildVimPluginFrom2Nix {
     pname = "Coqtail";
-    version = "2022-09-16";
+    version = "2022-09-30";
```

Comment:
why are vim plugin updates are in this PR ?

Reply:
Sorry about that push, rebased to a wrong branch, fix that in a min.